bsip-0086.md

```diff
@@ -0,0 +1,115 @@
+    BSIP: 0086
+    Title: Share market fees to the network
+    Author: Abit More <https://github.com/abitmore>
+    Status: Draft
+    Type: Protocol
+    Created: 2019-11-29
+    Discussion: https://github.com/bitshares/bsips/issues/194
+    Worker: TBD
+
+# Abstract
+
+This BSIP proposes a protocol change: when every trade happens, if there are
+market fees generated, a portion (E.G. 5% or 10%) of the market fees goes to
+the network.
+
+# Motivation
+
+The network needs more tools to generate income to support its development.
+
+# Rationale
+
+A major activity in the network is trading assets. The asset owners make
+profits in the form of market trading fees. Currently, the main tools for the
+network to generate income from asset trading activities are order creation
+fee and order cancellation fee. It seems reasonable that the asset owners
+share some profits (a part of market fees) to the network to support its
+development.
+
+The cut of market fees can go to committee-account's vesting balances.
+As a supporting measure, committee-account should be exempted from
+white-listing restrictions, so that it is able to claim the vesting balances
+and sell them for core token or other tokens to pay for development later.
+
+# Specifications
+
+## Protocol upgrade
+
+A time will need to be scheduled for applying the change. In this document,
+terms "before the protocol upgrade", "at the protocol upgrade" and "after
+the protocol upgrade" may or may not be used to indicate things happen before
```

Comment:
> may or may not be

I don't see how "may not" is helpful for specifications.  Might it be better to change this phrase to "will be"?

Reply:
Then also check that the wording is consistently used.

Reply:
It's for easier to copy & paste.

Reply:
Although I think it would be better to remove the "or may not", I think that the idea is clear enough in its current form.